src/dmd/statementsem.d

```diff
@@ -3308,6 +3308,16 @@ private extern (C++) final class StatementSemanticVisitor : Visitor
                     rs.exp = rs.exp.expressionSemantic(sc);
                 }
 
+                // @@@DEPRECATED_2.098@@@
+                // End deprecation cycle after 2.108.
+                uint olderrors = global.startGagging();
+                if (discardValue(rs.exp))
+                {
+                    //errors = true;
+                }
```

Comment:
Is this empty if really necessary?

Reply:
Uncomment when deprecation is over to turn it into an error.

Reply:
This should be noted in the source code